### DISCUSSION TITLE
Using $parent.mymethod and wire:loading does not work

### DISCUSSION THREAD

**Discussion by 7ammer** [livewire/livewire#7235] [7. Bugs (archived)]
### Livewire version

3.1.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Implement a nested component similar to documented [example](https://livewire.laravel.com/docs/nesting#directly-accessing-the-parent-from-the-child) and add `wire:loading` like so:

```
<div>
    <span>{{ $todo->content }}</span>
 
    <button wire:click="$parent.remove({{ $todo->id }})">Remove</button>
    
    <div wire:loading wire:target="$parent.remove({{ $todo->id }})">Loading</div>
</div>
```
2. Notice no "Loading" appears when button is clicked.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by tonsoflaz2** [comment]
Just want to second, upvote this.

The use case seems very straight forward: I have a list of components. I have a delete button on each nested component Item. 

The delete function exists on the parent function, so that the list can be re-drawn once the item is deleted.

Parent component:
```
@foreach ($items as $item)
    <livewire:item :key="$item->id" :$item />
@endforeach
```

Each nested component:
```
<div wire:loading.class="opacity-50">
    <i class="fa fa-close"
        wire:click="$parent.deleteItem('{{ $item->id }}')" 
       wire:confirm="Are you sure you want to delete this item?"></i>
    {{ $item->text }}
</div>
``` 

But the wire:loading.class="opacity-50" doesn't recognize this as a loading state for the nested component.

It seems like a very common use case: i am doing something with one item of the list, i want to indicate something is happening to the user before the item disappears.

Thank you in advance for any advice/discussion.

**Comment by saloeric** [comment]
same problem here

**Comment by danie-ramdhani** [comment]
I think you can't use `wire:loading` with `$parent` object because it's different component

**Comment by pbourdet** [comment]
Problem still here on a freshly installed livewire app. 

If anyone has a clean, working solution, I'm all ear 🙃
